### PR TITLE
Cut 0.15.0 release.

### PR DESCRIPTION
**Pull request by dhermes**
/cc @waprin

---

```
$ git log -1 --pretty=%H
b27907be562e81a10abe258ee32d039146e65768
$ git log 0.14.0..HEAD | grep 'Merge pull request ' |  awk -F ' ' '{print $4}' | sort
```

yields

#1772
#1787
#1802
#1804
#1805
#1808
#1810
#1820
#1823
#1825
#1831
#1834

### CODE REVIEW

**Comment by dhermes** [comment]
## Translate

- Added basic implementation (#1772)

## Datastore

- Fix bug where `limit` was not being reduced in paging queries (#1787)

## Logging

- Remove project ID validation when unmarshalling log entries (#1808)
- Bugfix: Remove metric and sink name validation (#1825)
- Beginning process to allow gRPC implementation side-by-side with HTTP (#1820, #1823)

## Bigtable

- Make the `Client` context manager return a valid context manager on entry (#1810)

## BigQuery

- Adding "dry run" for queries (#1834)

------

Admin type PRs not in release: #1802, #1804, #1805, #1831

File: setup.py [hunk 1]
```diff
@@ -25,7 +25,7 @@
 
 setup(
     name='gcloud',
-    version='0.14.0',
+    version='0.14.1',
```
Comment (tseaver) [inline setup.py]: Should be `0.15.0` (we've added a new API).

Comment (dhermes) [inline setup.py]: Good call. I saw the tiny number of commits so just instinctively did a minor version bump.

**Comment by tseaver** [comment]
LGTM

**Comment by dhermes** [comment]
FYI @waprin you are all good to update the docs you were working on in #1807

**Pull request merged**